internal/cmd/db_passwd.go

```diff
@@ -54,7 +54,7 @@ var changePasswordCmd = &cobra.Command{
 
 		bar := prompt.Spinner("Changing password...")
 		defer bar.Stop()
-		client, err = createTursoClient()
+		client, err = createTursoClientFromAccessToken(false)
```

Comment:
I'm wondering why we can't reuse the client from line 29

Reply:
Maybe the idea when it was created was to handle the error that wasn't handled in the first one.
https://github.com/chiselstrike/turso-cli/commit/adde529ea5e9e35ca7bbbc53f8c8f784112fe446

I tested it now removing the second client and it works.
I think it is safe to remove it, right?

Reply:
if the test passes then it should be fine.